src/styles/main.css

```diff
@@ -19,3 +19,49 @@ a:hover {
 .content ul {
   @apply my-6;
 }
+
+/* Show nav bar on scroll */
+#navbar {
```

Comment:
If the main.css file gets to get larger I'd say lets create a navBar css module and move this code there